style/buttons.less

```diff
@@ -0,0 +1,20 @@
+@btn-border-radius: 4px;
+@primary-background: #f47642;
+@primary-hover-background: #fa804d;
+
+// Override the default text color from bootstrap-material-design
+.btn-primary,
+.btn-success { color: @white !important; }
```

Comment:
Actually, I don't know. Just moved this code from tutor.less to buttons.less. I'll make the change.